dev/io.openliberty.microprofile.openapi.2.0.internal/bnd.bnd

```diff
@@ -34,7 +34,8 @@ Export-Package: \
     io.openliberty.microprofile.openapi20.validation
 
 Private-Package: \
-    io.openliberty.microprofile.openapi20.cache
+    io.openliberty.microprofile.openapi20.cache,\
```

Comment:
I think that all these `openapi20.*` packages should be `openapi20.internal.*` because they are not API. Also makes it consistent with the previous version (`io.openliberty.microprofile.openapi.internal`).

Reply:
I agree, I'll make a more generalized cleanup issue which includes this, since there are a few other changes which should be made which will affect lots of files.

Reply:
Raised #18790